website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -59,6 +70,18 @@ Validating a form on submission can occur on the client or the server and can be
   <source src="/assets/patterns/form-patterns/validation-on-submit-example.mp4" />
 </video>
 
+### Delayed validation
+
+Delayed validation occurs on the client and refers to validating the field after a lapse in keystrokes and a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped input into the field and after the interval of time has expired, the field is validated without an `onblur` event occurring.
+
+This method can be invasive and result in unintended validation errors by assuming the user has completed filling out a field. Thus, we recommend implementing either of the previous methods prior to delayed validation for most forms.
+
+Consider a user entering their credit card information; they may repeatedly reference a physical card when inputting the card number, which can cause extended delays between keystrokes and result in displaying a validation error preemptively.
+
+<video controls loop width="100%">
+  <source src="/assets/patterns/form-patterns/delayed-validation-example.mp4" />
+</video>
```

Comment:
The current video looks almost identical to the on focus change video so it's hard to tell that this is an example specific to delayed validation. It may be more impactful to show an example of the scenario outlined above, e.g., showing a user enter the first 4 of their credit card number, exaggerated pause, then error before they can continue entering the number.

Reply:
I've updated the example to be a bit more clear and reference the actual example referenced!